### DISCUSSION TITLE
Created spack package for hyperqueue

### DISCUSSION THREAD

**Discussion by Nortamo** [It4innovations/hyperqueue#490] [Show and tell]
Hi,

I created a spack package for hyperqueue https://github.com/spack/spack/pull/32475.
We maintain most of our Supercomputer stacks using spack, so it's convenient to have a package even though HQ is straightforward to install standalone. I used the documentation (https://it4innovations.github.io/hyperqueue") as the package home page url, or is there some other site/url which would be more suitable?  

Let me know if you would like some one who's developing HQ to be added in the maintainer field (This just means that you will get notifications when there are PR affecting the package). 

I'll try to update the package with the latest version when you do new releases.

**Comment by Kobzol** [comment]
Hi, that's awesome, thank you very much! I think that the documentation is the correct home page URL, it also links to this repository.

You can add me (Jakub Beránek, berykubik@gmail.com) as the maintainer.
